mrmustard/physics/converters.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains the converter functions between different representations.
+"""
+from typing import Iterable, Union
+from mrmustard.physics.representations import Representation, Bargmann, Fock
+from mrmustard import math, settings
+from mrmustard.utils.typing import Matrix, Vector, Scalar
+import numpy as np
+
+
+import numpy as np
+import warnings
+
+
+def to_fock(rep: Representation, cutoffs: Union[int, Iterable[int]] = None):
+    r"""This functions takes a Representaion and returns a new Fock Representation object."""
+    if isinstance(rep, Bargmann):
+        shape_of_cutoffs_without_batch = len(rep.b[0])
```

Comment:
How about the following, to cut down the number of lines and improve readability?

```
len_cutoff = len(rep.b[0])

if not cutoffs:
    cutoffs = settings.AUTOCUTOFF_MAX_CUTOFF

cutoffs = (cutoffs,) * len_cutoff if isinstance(cutoffs, int) else cutoffs

if len_cutoffs != len(cutoffs):
    raise
```

Reply:
Also, this concept of `len_cutoff` comes up a bunch of times. For example, we may wanna use it when initializing a circuit component to validate that a given ``Wires`` object has the right number of wires for a given representation.
Should be add a property `n_wires` or a `__length__` to `Representation`, for convenience and clarity? Not sure how this would work for Fock though